docs/crocks/Either.md

```diff
@@ -0,0 +1,103 @@
+# Either
+
+`Either l r`
+
+Crock which takes a `l` and a `r`, or just a `r`, and acts accordingly:
+* taking only `r` will proceed as `Identity`
+* taking `l` and `r` will handle errors (or results defined as `Left`) with `l` and results with `r` (or results defined as `Right`)
+
+The "`Either`" crock creates disjunction with a linear data flow.
+
+```js
+// Imperative flow
+const renderPage = user => {
+  const isAuthenticated = user ? true : false
+  return isAuthenticated ? "Home Page" : "Login Page"
+}
+
+// Declarative flow
+const isAuthenticated = u => u ? Either.Right(u) : Either.Left("Login Page")
+
+const renderPage = user =>
+  isAuthenticated(user)
+  .map(u => "Home Page")
+  .value()
+```
+
+`Either` exposes these constructors and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Left`](#left), [`Right`](#right), [`of`](#of) | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Left
+
+`Either m => l -> m l _`
+
+### Right
+
+`Either m => r -> m _ r`
+
+### of
+
+`Either m => a -> m _ a`
+
+## Instances
+
+### inspect
+
+`() => String`
+
+### either
+
+`Either m => m l r ~> (l -> c) -> (r -> d) -> m l d`
+
+### value
+
+`Either m => m l r ~> () => a`
```

Comment:
Wow, it is good to see these defs. It is amazing what you discover 

Now I realize that this is not valid and that `Either` and `Maybe` cannot/should not implement `value`. I am going to add an Issue for the `0.3.0` release to remove `value` from `Either` and `Maybe`. But keep it in the docs for now.